app/agent/activities/mutex/LICENSE

```diff
@@ -0,0 +1,23 @@
+Temporal TypeScript Samples
```

Comment:
Should the licence description in the README be updated?

Reply:
We already have a `unless noted otherwise` statement - I wouldn't list every single exception in the main README.